dff/script/extras/conditions/utils.py

```diff
@@ -0,0 +1,55 @@
+"""
+Utils
+------
+
+This module contains helpful constants, magic variables and types that help parse API responses.
+"""
+import re
+from typing import Optional, List
+
+from pydantic import BaseModel
+
+STATUS_UNAVAILABLE = 503
+STATUS_SUCCESS = 200
```

Comment:
I think these constants can be found already defained in `requests` and/or `httpx` packages.